Apps/US/HybridGP_US/app/src/Codeunits/HybridGPUSInstall.Codeunit.al

```diff
@@ -0,0 +1,61 @@
+namespace Microsoft.DataMigration.GP;
+
+using Microsoft.DataMigration;
+
+codeunit 42007 "Hybrid GP US Install"
+{
+    Access = Internal;
+    Subtype = Install;
+
+    trigger OnInstallAppPerDatabase()
+    begin
+        RegisterValidator();
+        RegisterTests();
+    end;
+
+    local procedure RegisterValidator()
```

Comment:
This should not be necessary. The better approach is the interface and an enum. If we have an extensible enum, each of the extension can add its own.

We could also raise an event and populate the list of interfaces, or insert on the fly. The issue with OnInstall is that it is happening once, users cannot reinstall the app.